src/GovUk.Education.ExploreEducationStatistics.Content.Services/PublicationService.cs

```diff
@@ -43,16 +37,11 @@ public class PublicationService : IPublicationService
                 .Include(p => p.Topic)
                 .ThenInclude(topic => topic.Theme)
                 .Where(p => p.Slug == publicationSlug))
-            // NOTE: BlobCache won't cache result if GetLatestRelease returns an Either.IsLeft - i.e. if there is no latestRelease
             .OnSuccessCombineWith(GetLatestRelease)
-            .OnSuccess(tuple =>
+            .OnSuccess(async tuple =>
             {
                 var (publication, latestRelease) = tuple;
-                var publicationViewModel = _mapper.Map<PublicationViewModel>(publication);
-                publicationViewModel.LatestReleaseId = latestRelease.Id;
-                publicationViewModel.IsSuperseded = IsSuperseded(publication);
-                publicationViewModel.Releases = ListPublishedReleases(publication);
-                return publicationViewModel;
+                return await BuildPublicationViewModel(publication, latestRelease);
```

Comment:
Why did you decide to do away with AutoMapper here out of interest?  Not that I'm complaining :D

Reply:
I thought I was going to need to copy the AutoMapper configuration from `Content.Services.Mappings.MappingProfiles` for `PublicationViewModel` to Admin and Publisher, and thought that would get extremely confusing, especially in the case of Admin where it's got it's own Admin `PublicationViewModel` (in fact it's got an additional one already for the manage content page!) and mappings for those already.

I can't remember now if I saw an error about the missing mappings and then decided on this approach, or if I just imagined this based on a change to Publisher's MappingProfiles in https://github.com/dfe-analytical-services/explore-education-statistics/pull/3463.

It's possible that I didn't need to make this change if the `Content.Services.Mappings.MappingProfiles` are loaded by Admin and Publisher but thought moving away from AutoMapper also makes it clearer how the view modesl are built.